harpy/_validations.py

```
@@ -73,98 +73,104 @@
             if not line:
                 break
-            if line == "\n":
-                break
-            # split the line by whitespace
-            rowvals = line.rstrip().split()
-            rowlen = len(rowvals)
-            if rowlen != 6:
-                badrows.append(row)
-                badlens.append(rowlen)
-            elif len(badrows) > 0:
+            if line is "\n":
```

Ruff

Use `==` to compare constant literals

Replace `is` with `==`

(F632)